src/components/BookList.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const BookList = () => (
+  <form className="newbook-form">
+    <input type="text" id="inputText" placeholder="Book Name" />
+    <input type="text" className="author-input" placeholder="Author Name" name="author" id="author" />
+    <button type="button" className="add-book">Add Book</button>
+  </form>
+);
```

Comment:
- [ ] You created the add book form in the `BookList` component, This component should called `AddBook`, and in `BookList` please use the array of books in it that is available in `Books`, map through the array, and render the `Books` component, then import the BookList in you `App`. Please make sure to fix this issue as requested by the previous reviewer.

Reply:
@thecodechaser , please I need your help
I am my extra review and it is Saturday